docs/script to save certificates.md

```diff
@@ -0,0 +1,42 @@
+# Scripts to save certificates
```

Comment:
please change the heading of the document to "Backup and Restore SSL Letsencrypt certificates"